.travis.yml

```diff
@@ -1,15 +1,19 @@
 # whitelist
 branches:
   only:
+      - master
       - tickets/DM-5218
 
-sudo: required
+sudo: false 
```

Comment:
Using `sudo: false` allow additional travis feature like caching: https://docs.travis-ci.com/user/workers/container-based-infrastructure/
And CI currently passes, so I believe Docker is compliant with it.